draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1664,8 +1666,9 @@ Implementations MAY implement a shared knowledge scheme introduced in {{distribu
 ## Unguessability {#unguessability}
 Implementations SHOULD utilize a cryptographically secure pseudo-random number
 generator (CSPRNG) to provide values that are both difficult to predict ("unguessable")
-and have a low likelihood of collision ("unique").
-Care SHOULD be taken to ensure the CSPRNG state is properly reseeded upon
+and have a low likelihood of collision ("unique").  The exception is when a
+suitable CSPRNG is unavailable in the execution environment.
+Take care to ensure the CSPRNG state is properly reseeded upon
```

Comment:
I think stating the intended exception here (no CSPRNG available) makes sense and clearly justifies the use of SHOULD.